packages/react-native-payments/package.json

```diff
@@ -76,7 +81,10 @@
     },
     "gitHead": "b5608910319390f9773a9d42c3cc828e8e8a1d95",
     "dependencies": {
+        "@expo/config-plugins": "8.0.10",
         "@rnw-community/shared": "workspace:*",
+        "fs": "0.0.1-security",
+        "path": "^0.12.7",
```

Comment:
This dependency can be omitted as a part of nodejs

Reply:
done